src/endpoints/user.rs

```diff
@@ -7,20 +7,31 @@ use chrono::{DateTime, Utc};
 /// Gets information about a user
 /// https://api.cloudflare.com/#user-user-details
 
+#[derive(Deserialize, Serialize, Debug, Clone, PartialEq, Eq)]
+pub struct Organization {
+    id: String,
+    name: String,
+    status: String, // Whether or not the user is a member of the organization or has an inivitation pending
+    permissions: Vec<String>, // Access permissions for this User
```

Comment:
Feels weird that permissions is a vector of Strings. Should probably be a vector of Permissions.

Reply:
in terms of like an Enum called Permissions? I think I agree, but what is tricky about that is the list is growing and not under our control. We'd have to keep track of the existing list of permissions managed by the API team and keep in lockstep to prevent breakage. Not 100% opposed, but just pointing out the additional overhead of maintaining that enum.

Reply:
it also feels weird that these permissions are part of the organization struct but i think i just don't get this api.